### DISCUSSION TITLE
i18n] [message-box] ElMessageBox 使用VNode时，不支持 <I18nT />组件

### DISCUSSION THREAD

**Discussion by whx110css** [element-plus/element-plus#16993] [Q&A]
ElMessageBox({
    title: 'Message',
    message: () =>
      <  I18nT  keypath="aa.bb.cc" tag="span"   >
          <  span style="color: red"  >测试< /span >
        < /I18nT >
      })

<I18nT />无法转成正常的代码，无法正确进行国际化

**Comment by ghost** [comment]
const i18n = useI18n()
![i18n](https://github.com/element-plus/element-plus/assets/81557497/7bf31f40-179e-4b64-8b42-5b8854610e27)
在setup顶层获取i18n
在使用的部分手动设置i18n作用域
